how-to-contribute-to-workshops/how-to-contribute-to-workshops.md

```diff
@@ -0,0 +1,60 @@
+# How to Contribute to Workshops
+
+## Basic Requirements for Integrating Any Workshop
+
+For any workshop that you wish to contribute, ensure that it is sufficiently distinct from the workshops that Bit Project currently has. See <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization#step-1-workshop-proposal">Workshop Proposal</a> for the complete list of the workshops Bit Project currently has.
+
+Successful integration of any workshop involves the following main stages and sub-stages:
+
+* **Placing your work in GitBook**
+* **Stage 1: Preparation**
+  * Step 1: Workshop Proposal
+  * Step 2: Interactive Component Idea
+  * Step 3: Responsibilities Delegated
+* **Stage 2: Slides and Components**
+  * Step 1: Presentation Slides Outline
+  * Step 2: Completing Slides and Interactive Component
+* **Stage 3: Presentation**
+  * Step 1: Presentation Practice
+  * Step 2: Present to Bit & Organization
+
+For detailed information about each step and sub-step of the above workflow, see <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization">Joining as an Organization</a>.
+
+<h2 id = "Porting">Porting an Existing Workshop Into Our System
+
+If you wish to contribute an existing workshop, ensure that all requirements outlined in <a href = "https://about.bitproject.org/teams/developer-relations/joining-as-an-organization">Joining as an Organization</a> have been met and make edits to your workshop as neccessary. 
```

Comment:
There should be an example of an actual slide (that is not in our style already) corresponding to its equivalent markdown card.